src/lib.rs

```diff
@@ -64,6 +64,9 @@ pub use header::{IntoHeaderValue, XSpanIdString, X_SPAN_ID};
 #[cfg(feature = "multipart")]
 pub mod multipart;
 
+pub mod one_any_of;
```

Comment:
Any particular reason to export the module, beyond consistency?

Reply:
No, just consistency.  Happy to not export it if you prefer?

Reply:
Yes, I don't think we need to expose the internal structure.

Reply:
No longer exposed.